SectionedSolidHorizontal-LinearDirectrix-1/UT-SectionedSolidHorizontal-LinearDirectrix-VerticalElement-1.ifc

```diff
@@ -0,0 +1,127 @@
+#16= IFCSIUNIT(*,.MASSUNIT.,.KILO.,.GRAM.);
+#17= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#18= IFCMEASUREWITHUNIT(IFCPLANEANGLEMEASURE(0.0174532925199433),#17);
+#19= IFCDIMENSIONALEXPONENTS(0,0,0,0,0,0,0);
+#20= IFCCONVERSIONBASEDUNIT(#19,.PLANEANGLEUNIT.,'DEGREE',#18);
+#22= IFCAXIS2PLACEMENT3D(#25,$,$);
+#25= IFCCARTESIANPOINT((0.,0.,0.));
+#27= IFCPROJECT('1VWTA2AQn4bezaJ7736V_D',#11,'Bridge 5',$,$,$,$,(#60),
+#37);
+#37= IFCUNITASSIGNMENT((#13,#14,#15,#16,#20));
+#39= IFCBRIDGE('2qX8Zx6Lv4oxQKYStwzyUq',#11,'Bridge5',$,'',#53,$,$,
+.ELEMENT.,.USERDEFINED.);
+#50= IFCAXIS2PLACEMENT3D(#51,$,$);
+#51= IFCCARTESIANPOINT((0.,0.,0.));
+#53= IFCLOCALPLACEMENT($,#50);
+#56= IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Body','Model',*,*,*,*,#60,$,
+.MODEL_VIEW.,$);
+#60= IFCGEOMETRICREPRESENTATIONCONTEXT($,'Model',3,1.E-5,#22,$);
+#63= IFCAXIS2PLACEMENT3D(#64,#66,#68);
+#64= IFCCARTESIANPOINT((890.5,52.2,-130.5));
+#66= IFCDIRECTION((0.,1.,0.));
+#68= IFCDIRECTION((0.,0.,1.));
+#70= IFCLOCALPLACEMENT(#53,#63);
+#71= IFCPRODUCTDEFINITIONSHAPE('Explicit Shape',$,(#159));
+#75= IFCPOLYLINE((#77,#79));
+#77= IFCCARTESIANPOINT((0.,0.,0.));
+#79= IFCCARTESIANPOINT((86.,0.,0.));
+#81= IFCSECTIONEDSOLIDHORIZONTAL(#75,(#83,#111),(#138,#144),.F.);
+#83= IFCARBITRARYCLOSEDPROFILEDEF(.AREA.,'',#86);
+#86= IFCCOMPOSITECURVE((#97,#109),$);
+#88= IFCTRIMMEDCURVE(#96,(IFCPARAMETERVALUE(3.50823878194633E-15)),(
+IFCPARAMETERVALUE(180.)),.T.,.PARAMETER.);
+#91= IFCAXIS2PLACEMENT2D(#92,#94);
+#92= IFCCARTESIANPOINT((-17.5,0.));
+#94= IFCDIRECTION((1.,0.));
+#96= IFCCIRCLE(#91,17.5);
+#97= IFCCOMPOSITECURVESEGMENT(.CONTINUOUS.,.T.,#88);
+#100= IFCTRIMMEDCURVE(#108,(IFCPARAMETERVALUE(0.)),(IFCPARAMETERVALUE(
+180.)),.T.,.PARAMETER.);
+#103= IFCAXIS2PLACEMENT2D(#104,#106);
+#104= IFCCARTESIANPOINT((-17.5,0.));
+#106= IFCDIRECTION((-1.,-1.22464679914735E-16));
+#108= IFCCIRCLE(#103,17.5);
+#109= IFCCOMPOSITECURVESEGMENT(.CONTINUOUS.,.T.,#100);
+#111= IFCARBITRARYCLOSEDPROFILEDEF(.AREA.,'',#112);
+#112= IFCCOMPOSITECURVE((#123,#134),$);
+#114= IFCTRIMMEDCURVE(#122,(IFCPARAMETERVALUE(3.50823878194633E-15)),(
+IFCPARAMETERVALUE(180.)),.T.,.PARAMETER.);
+#117= IFCAXIS2PLACEMENT2D(#118,#120);
+#118= IFCCARTESIANPOINT((-17.5,0.));
+#120= IFCDIRECTION((1.,0.));
+#122= IFCCIRCLE(#117,17.5);
+#123= IFCCOMPOSITECURVESEGMENT(.CONTINUOUS.,.T.,#114);
+#125= IFCTRIMMEDCURVE(#133,(IFCPARAMETERVALUE(0.)),(IFCPARAMETERVALUE(
+180.)),.T.,.PARAMETER.);
+#128= IFCAXIS2PLACEMENT2D(#129,#131);
+#129= IFCCARTESIANPOINT((-17.5,0.));
+#131= IFCDIRECTION((-1.,-1.22464679914735E-16));
+#133= IFCCIRCLE(#128,17.5);
+#134= IFCCOMPOSITECURVESEGMENT(.CONTINUOUS.,.T.,#125);
+#137= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(0.),$,$,$
+,#75);
+#138= IFCAXIS2PLACEMENTLINEAR(#137,#139,#141);
+#139= IFCDIRECTION((0.,0.,1.));
+#141= IFCDIRECTION((1.,-0.,-0.));
+#143= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(8600.),$,
+$,$,#75);
+#144= IFCAXIS2PLACEMENTLINEAR(#143,#145,#147);
```

Comment:
The `IfcPolyline` being the directrix, goes from (0,0) to (86,0); however the `IfcSectionedSolidHorizontal` goes until 8600. This needs revision - the units missmatch.